src/igdb.cpp

```diff
@@ -271,30 +271,43 @@ void Igdb::getRating(GameEntry &game) {
     }
 }
 
-QList<QString> Igdb::getSearchNames(const QFileInfo &info) {
-    QString baseName = info.completeBaseName();
+QList<QString> Igdb::getSearchNames(const QFileInfo &info, QString &debug) {
+    QString baseName = info.baseName();
```

Comment:
Same here: Use `QString baseName = info.completeBaseName();` to avoid regression.